moveit_ros/planning_interface/test/python_move_group.test

```diff
@@ -1,5 +1,9 @@
+<!-- -*- mode: XML -*- -->
```

Comment:
I'll merge this as is, but what is up with this header?
The standard header for xml is `<?xml version="1.0"?>`.
Does your IDE not recognize this one or did I miss a change in preferred format?

Reply:
I've used this for a long time and emacs does properly format the file when its included. Not sure where it came from though, but its in my templates

Reply:
That's just a header for emacs to "announce" the file mode for emacs. Not related to XML.